cmd/system-probe/modules/network_tracer_test.go

```diff
@@ -59,7 +59,7 @@ func TestDecode(t *testing.T) {
 	}
 
 	marshaller := encoding.GetMarshaler(encoding.ContentTypeJSON)
-	connectionsModeler := encoding.InitConnectionsModeler(in)
+	connectionsModeler := encoding.NewConnectionsModeler(in)
```

Comment:
I think you need to call the new cleaning function (whatever it ends up being named) with a `defer`.

Reply:
added